apps/challenges/models.py

```diff
@@ -504,6 +504,11 @@ class ChallengeConfiguration(TimeStampedModel):
         null=True,
         blank=True,
     )
+    code_upload_env_output_file = models.FileField(
+        upload_to=RandomFileName("zip_configuration_files/challenge_zip"),
+        null=True,
+        blank=True,
+    )
```

Comment:
Can you remind me why we have this separate file? Shouldn;t there be just 1 environment log file for each submission? Why are you adding it in challenge config? Please remove this field I don't think we need this field

Reply:
Oh, I see what you mean. I'll remove that,